recipes/catch2/2.x.x/CMakeLists.txt

```diff
@@ -7,4 +7,10 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 include(conanbuildinfo.cmake)
 conan_basic_setup()
 
+option(enable_benchmark "Enable benchmark" OFF)
+
+if (enable_benchmark)
+    add_definitions(-DCATCH_CONFIG_ENABLE_BENCHMARKING)
```

Comment:
This should be set in the main recipe under package_info no here because it will not propagate to the consumer (it should not be in the test package when built)